<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InteractiveTutorialDialog</class>
 <widget class="QDialog" name="InteractiveTutorialDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog

{
background: rgb(255, 255, 255);
}
QLabel{
color: black;
}</string>
  </property>
  <widget class="QLabel" name="gateLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>501</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLabel {
    border: 5px solid #dbb684;
    border-radius: 10px;
    background-color: #edd5b4;
    color: black;
    font: bold 24px;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="continueButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>630</y>
     <width>153</width>
     <height>70</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 5px solid #2A7F46;
    border-radius: 10px;
    background-color: #34A853;
    color: white;
    font: bold 24px;
    padding: 10px;
    min-width: 30px;
    min-height: 40px;
}

QPushButton:hover {
    background-color: #2C974B;
}

QPushButton:pressed {
    background-color: #278040;
}</string>
   </property>
   <property name="text">
    <string>CONTINUE</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>500</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="outputLabel">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>300</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
                               border: 5px solid #000000;
                               border-radius: 10px;
                               background-color: #ff9c9c;
                               color: black;
                               font: bold 24px;
                               padding: 10px;
                               min-width: 40px;
                               min-height: 40px;
text-align: center;
                           }</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="descriptionLabel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>0</y>
     <width>401</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel 

{
	
	font: 11.5pt &quot;Courier&quot;;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="robotImage">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>380</y>
     <width>271</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Images.qrc">:/robot/robot_and.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>gateLabel</zorder>
  <zorder>continueButton</zorder>
  <zorder>imageLabel</zorder>
  <zorder>outputLabel</zorder>
  <zorder>robotImage</zorder>
  <zorder>descriptionLabel</zorder>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
